CONTRIBUTING.md

```diff
@@ -1,5 +1,6 @@
 # Contributing to the Self Hosted Show Wiki
 
+See the [Beginners Contributing Guide](/basics/beginners-contributing-guide/) for step by step instructions on how to add content.
```

Comment:
This should be a fully resolved link (domain and all), so it works when viewed on GitHub too!

Reply:
also easy fix and ya i was not sure what it needed to be thanks.